app/assets/sass/customize.scss

```diff
@@ -0,0 +1,6 @@
+/* Within the two-thirds containers the image should keep the full width */
```

Comment:
Could we move this under `app/assets/sass/components/_images.scss` (so as to make/mark it as Images component specific change/customisation) and update it's import on `app/assets/sass/main.scss` ?